File: prob5.js
// https://projecteuler.net/problem=5
// Completed 12/29/2021

let currentNumber = 1;
let dividend;
let dividendStatus = false;

while(dividendStatus === false){
  dividendStatus = true;
  for(let i = 1; i <= 20; i++){
    if(currentNumber % i === 0){
      dividendStatus = true;
    } else if (currentNumber % i !== 0){
      dividendStatus = false;
      i = 20;
    }
  }
  if(dividendStatus === true){
    dividend = currentNumber;
  }
  currentNumber++;
}

console.log(dividend);

File: prob5.py
# https://projecteuler.net/problem=5
# Completed 12/29/2021

def rangeDivisible(num, first, last):
  for i in range(first, last):
    if num % i != 0:
      return False
  return True

num = 0
potential = 20
  
while num == 0:
  if rangeDivisible(potential, 1, 21):
    num = potential
  potential += 1
  
print(num)